scripts/install-worker.sh

```diff
@@ -119,50 +122,42 @@ sudo mkdir -p /etc/eks
 sudo mv $TEMPLATE_DIR/iptables-restore.service /etc/eks/iptables-restore.service
 
 ################################################################################
-### Docker #####################################################################
+### installing updated cli #####################################################
+### https://docs.aws.amazon.com/cli/v1/userguide/install-linux.html#install-linux-bundled
 ################################################################################
 
-sudo yum install -y device-mapper-persistent-data lvm2
-
-INSTALL_DOCKER="${INSTALL_DOCKER:-true}"
-if [[ "$INSTALL_DOCKER" == "true" ]]; then
-    sudo amazon-linux-extras enable docker
-    sudo groupadd -og 1950 docker
-    sudo useradd --gid $(getent group docker | cut -d: -f3) docker
-
-    # install runc and lock version
-    sudo yum install -y runc-${RUNC_VERSION}
-    sudo yum versionlock runc-*
-
-    # install containerd and lock version
-    sudo yum install -y containerd-${CONTAINERD_VERSION}
-    sudo yum versionlock containerd-*
-
-    # install docker and lock version
-    sudo yum install -y docker-${DOCKER_VERSION}*
-    sudo yum versionlock docker-*
-    sudo usermod -aG docker $USER
-
-    # Remove all options from sysconfig docker.
-    sudo sed -i '/OPTIONS/d' /etc/sysconfig/docker
-
-    sudo mkdir -p /etc/docker
-    sudo mv $TEMPLATE_DIR/docker-daemon.json /etc/docker/daemon.json
-    sudo chown root:root /etc/docker/daemon.json
-
-    # Enable docker daemon to start on boot.
-    sudo systemctl daemon-reload
+if [[ "$BINARY_BUCKET_REGION" != "us-iso-east-1" && "$BINARY_BUCKET_REGION" != "us-isob-east-1" ]]; then
+    echo "Installing updated AWS CLI v2"
+    AWSCLI_DIR=$(mktemp -d)
+    curl \
+      --silent \
+      --show-error \
+      --retry 10 \
+      --retry-delay 1 \
+      -L "https://awscli.amazonaws.com/awscli-exe-linux-${MACHINE}.zip" -o "${AWSCLI_DIR}/awscliv2.zip"
+    unzip -q "${AWSCLI_DIR}/awscliv2.zip" -d ${AWSCLI_DIR}
+    sudo "${AWSCLI_DIR}/aws/install"
+else
+    sudo yum install -y awscli
 fi
```

Comment:
nit... not sure if it would be a problem in the future.

for region `us-iso-east-1` and `us-isob-east-1`, command `aws` comes from `yum`. Both `aws` and `sudo aws` should works. But other regions, it will be different,

```
$ whoami
ec2-user

$ aws --version
aws-cli/2.8.4 Python/3.9.11 Linux/5.4.214-120.368.amzn2.x86_64 exe/x86_64.amzn.2 prompt/off

$ sudo aws --version
sudo: aws: command not found

$ which aws
/usr/local/bin/aws

$ sudo /usr/local/bin/aws --version
aws-cli/2.8.4 Python/3.9.11 Linux/5.4.214-120.368.amzn2.x86_64 exe/x86_64.amzn.2 prompt/off
```

Reply:
hm, this seems weird. The install script puts a symlink in `/usr/local/bin` so there's no reason `root` wouldn't see it, unless i'm missing something.